src/wirecloud/platform/iwidget/models.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2012-2016 CoNWeT Lab., Universidad Politécnica de Madrid
+
+# This file is part of Wirecloud.
+
+# Wirecloud is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# Wirecloud is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with Wirecloud.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from pydantic import BaseModel, Field, StringConstraints
+from typing import Annotated, Any, Optional
+from enum import Enum
+
+
+Id = str
+IntegerStr = Annotated[str, StringConstraints(pattern=r'^\d+$')]
+
+
+class DBWidgetConfigAnchor(BaseModel, use_enum_values=True):
```

Comment:
Hay que definir una clase que en lugar de heredar de BaseModel, hereda de Enum, y la usas en los demás sitios.
```suggestion
class DBWidgetConfigAnchor(Enum):
```